src/python_tools/notebooks/README.md

```diff
@@ -0,0 +1,6 @@
+# How to create new notebooks as .py
+- DO NOT CREATE NEW NOTEBOOKS IN JUPYTER NOTEBOOK WITH THE "NEW" MENU
+- Create an empty .py file in the notebooks directory
+- Open the empty .py file in Jupyter Notebook
+
+## DO NOT OPEN PULL REQUESTS WITH IPYNB NOTEBOOKS
```

Comment:
Is there a way to check in CI for if someone's making a PR w/ a Notebook? Can a `.gitignore` solve this too?

Reply:
added to gitignore, leaving this guide here because I still had to take a minute to figure out how to create a new notebook